### PR TITLE
Fix some small problems in mobile

### PR DESCRIPTION
**Pull request by mournfulCoroner**

### CODE REVIEW

File: tests/page-objects/dashboard/Description.ts [hunk 1]
```diff
@@ -19,29 +18,29 @@ export default class Description {
...
+        await this.page.waitForSelector(slct(DashMetaQa.SaveButton));
     }
 
     private async clickDescription() {
         await this.page.click(slct(COMMON_SELECTORS.ACTION_PANEL_DESCRIPTION_BTN));
     }
 
     private async opened() {
-        await this.page.$$(DashboardPage.selectors.dialogDashMeta);
+        await this.page.$$(`.${DashMetaQa.Dialog}`);
```
Comment (goshander) [inline tests/page-objects/dashboard/Description.ts:38]: `slct` helper?

Method `opened` has not return value and doesn`t wait any selector, is this correct?

File: tests/page-objects/dashboard/Description.ts [hunk 2]
```diff
@@ -19,29 +18,29 @@ export default class Description {
...
     }
 
     private async closed() {
         await waitForCondition(async () => {
-            const elements = await this.page.$$(DashboardPage.selectors.dialogDashMeta);
+            const elements = await this.page.$$(`.${DashMetaQa.Dialog}`);
```
Comment (goshander) [inline tests/page-objects/dashboard/Description.ts:43]: `slct` helper?

**Review by goshander — APPROVED** [review]

File: tests/page-objects/dashboard/Description.ts (hunk 2 above)
Comment (mournfulCoroner) [reply tests/page-objects/dashboard/Description.ts]: The original usage confused me. Added `slct`

File: tests/page-objects/dashboard/Description.ts (hunk 1 above)
Comment (mournfulCoroner) [reply tests/page-objects/dashboard/Description.ts]: You're right. Replaced by `waitForSelector`, as in other such cases of checking the opening of a dialog

**Pull request merged**